### DISCUSSION TITLE
What I need do to make lombok.jar works

### DISCUSSION THREAD

**Discussion by JohnWickzzz** [nvim-java/nvim-java#82] [Q&A] [answered]
I use the lombok in project, and I found the lombok.jar in **/.local/share/nvim/mason/packages/jdtls.
But in neovim, show that "the method getXX() is undefined for the type"
What configuration I need to set or update 🥺

**Comment by s1n7ax** [comment]
oh no.. completely forgot about lombok. That should work out of the box. I will fix this ASAP

**Comment by s1n7ax** [comment, marked as answer]
@buptlibo Hello. Please update the plugins. Make sure the nvim-java-core version is `1.0.1`. This issue was fixed by https://github.com/nvim-java/nvim-java-core/issues/47
![image](https://github.com/nvim-java/nvim-java/assets/18459807/f97191e2-1717-49d0-b7ef-230670a9acf5)

**Reply by JohnWickzzz** [reply]
I think it works, thank you👍
